src/libraries/periphery/MorphoBalancesLib.sol

```diff
@@ -45,7 +45,7 @@ library MorphoBalancesLib {
 
         uint256 elapsed = block.timestamp - market.lastUpdate;
 
-        if (elapsed != 0 && market.totalBorrowAssets != 0) {
+        if (elapsed != 0) {
```

Comment:
note that this change was not necessary, the code is correct without. But @Rubilmax finds it better like that, and I'm ok.

Reply:
>This change is required if the rate is non-zero while totalBorrowAssets is zero. Could such an IRM exist? I don't think it makes any sense, so we may prefer to skip the view call in this case and save some gas?

no it's not true, because in Blue

```solidity
uint256 interest = market.totalBorrowAssets.wMulDown(borrowRate.wTaylorCompounded(elapsed));
```

so the interest is always zero when there is no borrow

Reply:
Alright then I agree to revert this change ^^

Reply:
I think we should add a comment because it's not obvious at all why we're doing this here